kbase-extension/static/kbase/js/widgets/appWidgets2/input/floatInput.js

```diff
@@ -158,6 +158,18 @@ define([
                 initialControlValue = String(currentValue);
             }
 
+            function boundaryDiv(value, isMin) {
```

Comment:
Remove a little hard-to-read code / duplication. Next PR (or this one?) will move this to an accessory module.